File: server/php/src/Presentation/Models/GetPaymentByHostedCheckoutId/Response.php
<?php

namespace OnlinePayments\ExampleApp\Presentation\Models\GetPaymentByHostedCheckoutId;

use OnlinePayments\ExampleApp\Application\Interfaces\Responses\ResponseInterface;

final class Response implements ResponseInterface
{
    public ?string $status = null;

    public ?string $paymentId = null;

    public function toArray(): array
    {
        return [
            "status" => $this->status,
            "paymentId" => $this->paymentId
        ];
    }
}

File: server/ruby/app/presentation/models/get_payment_by_hosted_checkout_id/response.rb
module Presentation
  module Models
    module GetPaymentByHostedCheckoutId
      class Response
        attr_accessor :status, :payment_id

        def initialize(status: nil, payment_id: nil)
          @status = status
          @payment_id = payment_id
        end

        def as_json(*)
          { status: status, payment_id: payment_id }
        end
      end
    end
  end
end